src/test/java/uk/nhs/hee/trainee/details/service/impl/PersonalDetailsServiceImplTest.java

```diff
@@ -94,45 +128,77 @@ void shouldUpdateContactDetailsWhenTraineeIdFound() {
     assertThat("Unexpected personal details.", personalDetails.get(), is(expectedPersonalDetails));
```

Comment:
Should be refactored to use the new field values.

Reply:
Think I've addressed this in lines above now.